shotover-proxy/src/frame/cassandra.rs

```diff
@@ -224,10 +224,11 @@ impl CassandraFrame {
                                         .into_iter()
                                         .map(|row| {
                                             row.into_iter()
-                                                .map(|row_content| {
-                                                    MessageValue::Bytes(
-                                                        row_content.into_bytes().unwrap().into(),
-                                                    )
+                                                .map(|row_content| match row_content.into_bytes() {
```

Comment:
This seems familiar. Did we make another change somewhere else similar to this?

Reply:
I believe you attempted to make this change (or maybe something that was just similar?) in another PR and I asked it to be removed because I didnt yet understand the NO_METADATA scenario.